components/AirdropVerifier.vue

```diff
@@ -4,48 +4,77 @@
       'flex',
       'flex-col',
       'items-center',
+      'justify-center',
+      'px-[24px]',
     ]"
   >
     <img
-      :class="['my-[32px]', 'w-[300px]']"
+      :class="[
+        'hidden',
+        'my-[32px]',
+        'w-[300px]',
+        'lg:block',
+      ]"
       src="/images/airdrop/title_Checker.png"
     />
-    <!-- Subscribe -->
     <Label
-      :class="['text-medium-gray', 'my-[8px]']"
-      :text="address"
-      preset="p5"
+      :class="[
+        'my-[24px]',
+        'text-center',
+        'font-extrabold',
+        'text-like-green',
+        'lg:hidden',
+      ]"
+      :text="$t('AirDrop.label.checker')"
+      preset="h2"
+      align="center"
     />
```

Comment:
Can be refactored with the following?
https://github.com/likecoin/app-like-co/blob/a1d67dcf72e4f7f4438c096de72dc23f03368a59/components/AirdropLogin.vue#L21-L32